src/core/src/buffer.rs

```diff
@@ -48,19 +48,25 @@ impl Error for ViewError {
 bitflags!(
     /// Buffer usage flags.
     #[cfg_attr(feature = "serialize", derive(Serialize, Deserialize))]
-    pub flags Usage: u16 {
+    pub flags Usage: u32 {
```

Comment:
not sure why we'd need `u32` here just yet?

Reply:
Whoops, I was tired when I was writing that and thought I was going from u8 to u16!